Program/23bca284.py

```diff
@@ -0,0 +1,44 @@
+# Create table cricketer(cid, cname, match, run) and insert 8 players records. Print player records with average. and write this data into player.csv file. Do all this task from python.
+import sqlite3 as sq
+con=sq.connect("D:/23BCA284/sqlite3/database/cricket.db")
+c=con.cursor()
+# create the cricketer table¶
+c.execute("create table if not exists  cricketer(cid,cname,match,run)")
+# Insert player records into the table
+players = [
+    (1, 'Player1', 10, 150),
+    (2, 'Player2', 12, 180),
+    (3, 'Player3', 8, 200),
+    (4, 'Player4', 15, 130),
+    (5, 'Player5', 11, 170),
+    (6, 'Player6', 9, 160),
+    (7, 'Player7', 14, 140),
+    (8, 'Player8', 13, 190)
+]
+c.executemany('INSERT INTO cricketer (cid, cname, match, run) VALUES (?, ?, ?, ?)', players)
+con.commit()
+# fetch player data with average runs per match
+c.execute("select cname, match, run, (run * 1.0 / match) as average from cricketer")
+rec=c.fetchall()
+print(rec)
+# writing csv file
+import csv
+players = [

```

Comment:
Is there another way to insert records into the table instead of creating list "player" ??

Reply:
yes , we will insert record directly using writerow object , but it is time conjuming so, according to me using the list to insert record is a eassy way.